src/message-card/constants.ts

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+export const BACKGROUND_COLOR_TYPE = {
```

Comment:
`BACKGROUND_COLOR_TYPE` is bulky. Welcoming suggestions for alternate ways to name this constant.